google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py

```diff
@@ -105,6 +105,9 @@ class StreamingPullManager(object):
             ``projects/{project}/subscriptions/{subscription}``.
         flow_control (~google.cloud.pubsub_v1.types.FlowControl): The flow
             control settings.
+        use_legacy_flow_control (bool): Disables enforcing flow control settings
```

Comment:
I'd prefer disable_server_side_flow_control so it's right in the name. We can mention that it's legacy in the comment.

Reply:
Offline convo: we're using this name cross-platform and will try to deprecate it asap.